src/cascadia/TerminalSettingsModel/TerminalSettingsSerializationHelpers.h

```diff
@@ -165,10 +165,12 @@ JSON_ENUM_MAPPER(::winrt::Windows::UI::Xaml::ElementTheme)
 
 JSON_ENUM_MAPPER(::winrt::Microsoft::Terminal::Settings::Model::LaunchMode)
 {
-    JSON_MAPPINGS(3) = {
+    JSON_MAPPINGS(5) = {
         pair_type{ "default", ValueType::DefaultMode },
         pair_type{ "maximized", ValueType::MaximizedMode },
         pair_type{ "fullscreen", ValueType::FullscreenMode },
+        pair_type{ "focused", ValueType::FocusedMode },

```

Comment:
Man this is gonna seem like the most trivial nit, but maybe the string should just be `focus`, as in "I want the launch mode to be focus mode". I think if it's `focused`, then that makes me think "I want the terminal to launch and be focused", implying there's an `unfocused` for opening the terminal w/o focusing the window (which is _not_ something we plan on adding, certainly not in this property).

(& also change the below to `maximizedFocus`)